WordPress/src/main/java/org/wordpress/android/ui/CollapseFullScreenDialogFragment.java

```diff
@@ -520,4 +521,10 @@ public Builder setTitle(@StringRes int textId) {
             return this;
         }
     }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        mBinding = null
```

Comment:
Blocker (🚫): `;` is expected here, this is a Java file. Don't forget to build and test your code before pushing. 🤷